app-ios/tutanota/Sources/Sharing/SharingUtils.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import CryptoKit
+
+/// given a vcard, extract the value of the mandatory formatted name (FN) field or return a default of "contact"
+func extractFNfrom(vcard: String) -> String {
+  let fnRegex = try! NSRegularExpression(pattern: "^\\s*FN:(.*)$", options: .anchorsMatchLines.union(.caseInsensitive))
```

Comment:
do we still need `NSRegularExpression`?

Reply:
don't know, we want to keep it consistent with the rest of the app. we can make a maintenance task to modernize all usages of NSRegularExpression.